scripts/pso-collect-logs.sh

```diff
@@ -1,34 +1,60 @@
-if [[ $# != 0 ]]; then
-  echo -e "If kubeconfig is not configured please run: export KUBECONFIG=[kube-config-file]\n"
-  echo -e "Usage: *.sh"
-  exit
+FULL_MODE="false"
+while [ -n "$1" ]; do # while loop starts
+  case "$1" in
+
+  -full)
+    echo "-full option specified"
+    FULL_MODE="true";;
+
+  --help)
```

Comment:
two dashes here, but only 1 for the full option. Let's stick to 2 for both given the words are written in full, as per Linux standards. You only use a single dash for single-character switches, eg `-f` as opposed t o`--full`